redash/handlers/dashboards.py

```diff
@@ -25,6 +41,11 @@ class DashboardListResource(BaseResource):
     def get(self):
         dashboards = [d.to_dict() for d in models.Dashboard.all(self.current_org, self.current_user.groups, self.current_user)]
 
+        for dashboard in dashboards:
+            last_change = models.Change.get_latest(object_id=dashboard['id'], object_type=models.Dashboard.__name__)
+            if last_change:
+                dashboard['latest_version'] = last_change.id
```

Comment:
The current version of a dashboard should be the tracked object's property. It will save us from doing this lookup, but also will be used for the locking.